Makefile

```diff
@@ -268,7 +268,12 @@ set -e; \
 package=$(2)@$(3) ;\
 echo "Downloading $${package}" ;\
 rm -f $(1) || true ;\
-GOBIN=$(LOCALBIN) go install $${package} ;\
+GOBIN=$(LOCALBIN) \
+GOOS=$$(go env GOOS) \
+GOARCH=$$(go env GOARCH) \
+GO111MODULE=on \
```

Comment:
I don't think GOOS and GOARCH are needed, not sure about GO111MODULE. Maybe CGO_ENABLED=0 is good enough.

But if it is needed for sure, happy to merge it.